### DISCUSSION TITLE
New meeting will not post because of address not found

### DISCUSSION THREAD

**Discussion by vkay11518** [code4recovery/12-step-meeting-list#750] [Q&A] [answered]
Hello, my name is Vangie.  I'm new to this program and need help.  I'm having a problem adding another day to an existing meeting.  The program doesn't accept the address I type in and defaults back to just the city and town, zip, USA.  The error message says 'in person meetings need a specific address'.  Because there is a error, the 'update post' button is not available.  This meeting was already in the system on Tuesdays with the same address as I was trying to type, so is there a way to override this error message.  Thanks to anyone who has a fix for this.

**Comment by joshreisner** [comment]
Hi Vangie, what's the address you're trying to type? And what address does it revert to?

When I go to add a new meeting at an existing location, when I start typing the location name, eg "First Community Church" then I see a dropdown and I'm able to pick the meeting location, and it will supply the address for me. Is that not happening for you?

**Reply by joshreisner** [reply]
sorry that's happening to you @vkay11518! google geocoding is so good 99.99% of the time, but in this case it can't seem to find that address. we have a method of overriding addresses, but it means you'd have to make a small edit to your theme's functions.php file. is that something you're ok doing?

you would add this code somewhere (anywhere should be fine)

```
if (function_exists('tsml_custom_addresses')) {
    tsml_custom_addresses(array(
        '1 Church Plaza, Mora, NM 87732, USA' => array(
            'formatted_address' => '1 Church Plaza, Mora, NM 87732, USA',
            'city' => 'Mora',
            'latitude' => 35.9730664,
            'longitude' => -105.3301475,
            'approximate' => 'no',
        ),
    ));
}
```

**Comment by vkay11518** [comment, marked as answer]
Hi Josh,  it’s 1 Church Plaza, Mora, NM 87732.  After I enter that, it reverts to just Mora, NM 87732 USA and the error message comes up.  It says In person meetings need a specific address.  I don’t get a drop box when I start typing on that.  I do on some other entries.  It doesn’t seem to matter if I hit enter or tab after typing the address.  Vangie

**Reply by joshreisner** [reply]
(i edited your comment to remove your cell number, which got in there somehow)

**Reply by brianw-area24** [reply]
The St Gertude web site (https://stgertrudemora.org/) uses the address 8 County Road A033, Mora NM 87732. I'm wondering if this address might work?

**Reply by joshreisner** [reply]
wow yes that should work, thank you @brianw-area24 -- this is a preferable answer for two reasons: a) no code change b) should work with the meeting guide app (if that's a concern)

**Comment by vkay11518** [comment]
Thank you both for solving my issue.  I’m new to this and wouldn’t have felt comfortable changing code anywhere!  Vangie ***@***.***      From: Josh ReisnerSent: Wednesday, April 6, 2022 10:21 PMTo: code4recovery/12-step-meeting-listCc: vkay11518; MentionSubject: Re: [code4recovery/12-step-meeting-list] New meeting will not post because of address not found (Discussion #750) wow yes that should work, thank you @brianw-area24 -- this is a preferable answer for two reasons: a) no code change b) should work with the meeting guide app (if that's a concern)—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
